app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,176 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$light-gray: #f2f2f2;
+$white: #fff;
+$beach-blue: #5bc0de;
+
+body {
+  padding-top: 20px;
+}
+
+.index-background-header {
+  color: $white;
+  font-family: Circular,"Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-size: 5em;
+  font-weight: bold;
+  padding-top: 300px;
+  text-align: center;
+}
+
+.search-bar-container .well {
+  background: rgba(0, 0, 0, 0.1);
+  border-color: transparent;
+  bottom: 0;
+  left: 0;
+  margin-bottom: 0;
+  position: absolute;
+  right: 0;
+}
+
+.columns {
+  padding-top: 150px;
+}
+
+.index-containers {
+  margin-top: 100px;
+
+  * {
+    color: $white;
+  }
+
+  h1 {
+    color: inherit;
+  }
+
+  p {
+    color: inherit;
+  }
+}
+
+.beach-snapshot {
+  margin-bottom: 15px;
+
+  h1 {
+    margin-bottom: 3px;
+  }
+
+  span {
+    color: darken($light-gray, 30%);
+    padding-bottom: 100px;
+  }
+}
+
+.beach-snapshot-icon-container {
+  margin-top: 25px;
+}
+
+.center {
+  text-align: center;
+}
+
+.inline-block {
+  display: inline-block;
+}
+
+.amenity-name {
+  vertical-align: 9px;
+}
+
+.col-centered {
+  display: inline-block;
+  float: none;
+  text-align: center;
+}
+
+.section-title {
+  display: inline-block;
+  vertical-align: 20px;
+}
+
+footer {
+  background: $beach-blue;
+
+  a, span {
+    color: $white;
+    opacity: 0.8;
+  }
+
+  a:hover {
+    color: $white;
+    text-decoration: none;
+  }
+
+  h3 {
+    color: $white;
+  }
+}
+
+hr {
+  border-color: lighten(gray, 30%);
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.